docs/framework/wcf/add-service-reference-in-a-portable-subset-project.md

```diff
@@ -4,7 +4,8 @@ ms.date: "03/30/2017"
 ms.assetid: 61ccfe0f-a34b-40ca-8f5e-725fa1b8095e
 ---
 # Add Service Reference in a Portable Subset Project
-Portable subset projects enable .NET assembly programmers to maintain a single source tree and build system while still supporting multiple .NET implementations (desktop, Silverlight, Windows Phone, and XBOX). Portable subset projects only reference .NET portable libraries which are a .NET framework assembly that can be used on any .NET implementation.  
+
+Portable subset projects enable .NET assembly programmers to maintain a single source tree and build system while still supporting multiple .NET implementations (desktop, Silverlight, Windows Phone, and XBOX). Portable subset projects only reference .NET portable libraries which are a .NET Framework assembly that can be used on any .NET implementation.
```

Comment:
```suggestion
Portable subset projects enable .NET assembly programmers to maintain a single source tree and build system while still supporting multiple .NET implementations (desktop, Silverlight, Windows Phone, and XBOX). Portable subset projects only reference portable libraries that are .NET assemblies that can be used on any .NET implementation.
```

Reply:
Should XBOX be changed to Xbox ?
I think this is the correct brand name.

See Google search results:

![image](https://user-images.githubusercontent.com/31348972/71739780-1fe59100-2e63-11ea-9342-eb7b970a375e.png)